source/Octopus.Client/app.config

```diff
@@ -9,6 +9,6 @@
     </assemblyBinding>
   </runtime>
   <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2"/>
```

Comment:
Do we even need this?

Reply:
That's a good question. I am unsure to be honest. 👀

Reply:
We removed all these from other projects, but they are no longer targeting a mix of netfull + netcore

Reply:
I think we can remove the whole thing. The documentation only talks about "applications". This [blog post supports](https://stackoverflow.com/questions/21566528/what-happens-if-i-remove-the-auto-added-supportedruntime-element) that theory.

We don't even ship the config file:

![image](https://user-images.githubusercontent.com/1627582/79513228-a62a8980-8086-11ea-8f97-4c5a64024a26.png)